sklearn/cluster/hierarchical.py

```diff
@@ -239,7 +239,7 @@ def ward_tree(X, connectivity=None, n_clusters=None, return_distance=False):
         else:
             return children_, 1, n_samples, None
 
-    connectivity, n_components = _fix_connectivity(X, connectivity,
+    connectivity, n_connected_components = _fix_connectivity(X, connectivity,
                                                    affinity='euclidean')
```

Comment:
This indentation is incorrect now

Reply:
Fixed in commit [#3e0536a](https://github.com/scikit-learn/scikit-learn/pull/13427/commits/3e0536ae2157575e0af53a9124f6407f6cfd1439)